src/plugins/userIndicator/components/MissionStatusPopup.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="c-user-control-panel__component">
+    <div class="c-user-control-panel__header">
+      <div class="c-user-control-panel__title">Mission Status</div>
+      <button
+        aria-label="Close"
+        class="c-icon-button c-icon-button--sm t-close-btn icon-x"
+        @click.stop="onDismiss"
+      ></button>
+    </div>
+    <div class="c-ucp-mission-status">
+      <template v-for="action in missionActions" :key="action">
+        <label class="c-ucp-mission-status__label" :for="action">{{ action }}</label>
+        <div class="c-ucp-mission-status__widget" :class="getMissionActionStatusClass(action)">
+          {{ missionActionStatusOptions[missionActionStatusMap[action]]?.label }}
+        </div>
+        <div class="c-ucp-mission-status__select">
+          <select
+            :id="action"
+            v-model="missionActionStatusMap[action]"
+            name="setMissionActionStatus"
+            @change="onChangeStatus(action)"
+          >
+            <option
+              v-for="option in missionActionStatusOptions"
+              :key="option.key"
+              :value="option.key"
+            >
+              {{ option.label }}
+            </option>
+          </select>
+        </div>
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+import { inject, ref } from 'vue';
+
+import { useEventEmitter } from '../../../ui/composables/event';
+
+export default {
+  inject: ['openmct'],
+  emits: ['dismiss'],
+  async setup() {
+    const openmct = inject('openmct');
+    let missionActions = ref([]);
+    let missionActionStatusOptions = ref([]);
+    let missionActionStatusMap = ref({});
+
+    try {
+      // Listen for missionActionStatusChange events
+      useEventEmitter(openmct.user.status, 'missionActionStatusChange', ({ action, status }) => {
```

Comment:
The handler parameter signature does not match the values emitted by the StatusAPI here: 
https://github.com/nasa/openmct/pull/7418/files#diff-de4973d1b097ba7bca4bd6b61530a0edee5925fd20eb9fb537a1bd133d474119R348

Reply:
well, it works

Reply:
No it doesn't. There's a console error thrown and if you change the mission status in one browser window I don't see it reflected in a different window.
<img width="535" alt="image" src="https://github.com/nasa/openmct/assets/5247066/b0b14e90-a588-437e-96ed-a0754ee8b5fe">

Reply:
I'm still seeing this issue testing locally. Will check on the banner deployment and resolve if it's OK